src/layouts/template.tsx

```diff
@@ -139,9 +140,10 @@ export class Template {
   private EnableClientSideJavaScript(): ReactElement {
     return (
       <script
+        // eslint-disable-next-line react/no-danger
         dangerouslySetInnerHTML={{
           __html:
-            `document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');`,
+            'document.body.className = ((document.body.className) ? document.body.className + \' js-enabled\' : \'js-enabled\');',
```

Comment:
hmmm, we should check that classes are correct when this is built

Reply:
I agree. I seem to recall this may be problematic. This is the linter auto-fixing things.

Reply:
Tested in my environment and the `js-enabled` has been correctly applied 🎉